packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.mm

```diff
@@ -132,6 +132,20 @@ - (UIView *)viewWithModuleName:(NSString *)moduleName
   return [self viewWithModuleName:moduleName initialProperties:nil launchOptions:nil];
 }
 
+- (void)initializeReactHostWithLaunchOptions:(NSDictionary *)launchOptions
+{
+  if (_configuration.bridgelessEnabled) {
+    // Enable TurboModule interop by default in Bridgeless mode
+    RCTEnableTurboModuleInterop(YES);
+    RCTEnableTurboModuleInteropBridgeProxy(YES);
+
+    [self createReactHostIfNeeded:launchOptions];
+}
+
+  [self createBridgeIfNeeded:launchOptions];
+  [self createBridgeAdapterIfNeeded];
+}
```

Comment:
is almost identical to the code in the `viewWithModuleName:initialProperties:launchOptions:`. Can we avoid code duplication and find a way to reuse it?

Reply:
done